README.rst

```diff
@@ -171,6 +177,27 @@ We highly recommend trying both the Anaconda prompt, as well as the standard dos
 
 If you want to build the documentation, run: `conda install -y -c conda-forge sphinx_rtd_theme sphinx-gallery`
 
+Setting up your _pyspecdata configuration file
+----------------------------------------------
+Part of the pySpecData package is the datadir module, allowing the user to run the same code on 
+different machines - even thought he location of the raw spectral data might change. 
+This is controlled by the ``~/.pyspecdata`` or ``~/_pyspecdata`` config file. 
+in the key ``[General]`` there should be a value pair for ``data_directory = ``.
+The value should correspond either to your Rclone remote or you can simply copy and
+paste the location of your data on the machine.
+
+Rclone is a useful tool to get files off of the team's drive or whatever cloud storage you may use. 
+This enables you to store datafiles locally allowing the user to work offline if needed.
+Prior to setting up Rclone you will need to make a local file (in your home directory) 
+where your data files will be added to as called. Typically, the name of this folder corresponds
+to the name of the folder on the cloud (e.g. 'exp_data').
+
+To get set up with Rclone, download Rclone and follow the documentation which should include
```

Comment:
this should be a subsection of above -- look up an rst cheatsheet for how to look at levels of secitons